packages/features/ee/organizations/pages/settings/members.tsx

```diff
@@ -19,6 +23,15 @@ const MembersView = () => {
   const canLoggedInUserSeeMembers =
     (currentOrg?.isPrivate && isOrgAdminOrOwner) || isOrgAdminOrOwner || !currentOrg?.isPrivate;
 
+  useEffect(
+    function refactorMeWithoutEffect() {
```

Comment:
I know we have this pattern already somewhere in app but can we not simply remove this effect and have the same behaviour ?

Even when the component re-renders before the navigation completes, it is still asking us to go to the same page and thus it is `idempotent` in nature.